ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10-upgrade-from-eus-4.8.yaml

```diff
@@ -9,6 +22,10 @@ releases:
       stream: nightly
       version: "4.9"
   latest:
+    release:
```

Comment:
I think we need a comment here explaining why for future participants to understand the workaround.

Reply:
@jhou1  Unfortunately, we are not allowed to add comments in this yaml file. `make update` will remove all the comment lines.